client/modules/App/components/Overlay.jsx

```diff
@@ -68,17 +69,22 @@ class Overlay extends React.Component {
           >
             <header className="overlay__header">
               <h2 className="overlay__title">{title}</h2>
-              <div className="overlay__actions">
-                {actions}
-                <button
-                  className="overlay__close-button"
-                  onClick={this.close}
-                  aria-label={this.props.t('Overlay.AriaLabel', { title })}
-                >
-                  <ExitIcon focusable="false" aria-hidden="true" />
-                </button>
-              </div>
+              <MediaQuery query="(min-width: 768px)">
+                <div className="overlay__actions">{actions}</div>
+              </MediaQuery>
+              <button
+                className="overlay__close-button"
+                onClick={this.close}
+                aria-label={this.props.t('Overlay.AriaLabel', { title })}
+              >
+                <ExitIcon focusable="false" aria-hidden="true" />
+              </button>
             </header>
+            <MediaQuery query="(max-width: 767px)">
+              {actions && (
+                <div className="overlay__actions-mobile">{actions}</div>
```

Comment:
Let's a some left padding to this new CSS class.  This will align it with the QuickAddWrapper.

```
.overlay__actions-mobile {
  padding-left: #{24/ $base-font-size}rem;
}
```

(I know I said in the issue not to worry about padding, but seeing it in action...I've changed my mind 😝)

![image](https://github.com/processing/p5.js-web-editor/assets/28965286/e0b5ea9c-1c96-4dbe-9057-bd80fa64fbbf)

Reply:
so do you want me run `git revert 8c8dfa16c4ad9ddb7d9522ee9add70de9d5d6363` and push again..?

Reply:
or else do i need to do any changes?